static/app/utils/replays/hooks/useLogReplayDataLoaded.tsx

```diff
@@ -22,9 +22,12 @@ function useLogReplayDataLoaded({fetchError, fetching, projectSlug, replay}: Pro
     if (fetching || fetchError || !replay || !project) {
       return;
     }
-    const feErrorIds = replay.getReplay().error_ids || [];
-    const allErrors = replay.getRawErrors();
-    const beErrorCount = allErrors.filter(error => !feErrorIds.includes(error.id)).length;
+    const replayRecord = replay.getReplay();
+    const allErrors = replay.getErrorFrames();
+    const feErrorIds = replayRecord.error_ids || [];
+    const beErrorCount = allErrors.filter(
+      error => !feErrorIds.includes(error.data.eventId)
+    ).length;
```

Comment:
Here's the part where we take all the FE captured errors, and remove them from the full list, leaving the backend errors behind. 

If there's a better way to get a count of backend errors we can explore it.

Information that we have to use are:
- we know the project id/slug/platform of the replay
- platform might be undefined, for older projects
- the replay list page shows `count_errors` which should match `replayRecord.error_ids.length`
    - not all the errors in that list are guaranteed to exist
- the org might have only one project (think: laravel + vue)